release.nix

```diff
@@ -234,6 +303,9 @@ in
 
     grpc                       =       pkgs.grpc;
 
-    inherit pkgs config overlay;
+    inherit pkgs config overlay shell stack-env;
     inherit (pkgs) test-grpc-haskell;
+
+    inherit hsPkgsOverridden; # Function :: nixpkgs -> new haskellPackages
+    inherit (hsPkgsOverridden (nixpkgs {})) data-diverse proto3-wire proto3-suite;
```

Comment:
Do we need to re-export `data-diverse` / `proto3-wire` / `proto3-suite`?

Reply:
@Gabriella439 if someone is using the library without using the `overlay` it can be really useful to reach those already patched dependencies (all these 3 require patching).